cmd/config_init.go

```diff
@@ -28,7 +28,7 @@ override the options.`,
 		authMethod, auther := getAuthentication(cmd)
 
 		s := &settings.Settings{
-			Key:        generateRandomBytes(64), // 256 bit
```

Comment:
Well... This mustn't be removed because Settings.Key **is not** TLS Key.

Reply:
Please see #615. It will help this PR and a struct is better than an arbitrary map.

Reply:
Found a way not to use force and still use the server settings from DB merged with provided